cmd/build.go

```diff
@@ -47,11 +46,13 @@ var buildCmd = &cobra.Command{
 func init() {
 	Promu.AddCommand(buildCmd)
 
+	buildCmd.Flags().Bool("cgo", false, "Enable CGO")
```

Comment:
Actually, this is the `build` command, not the `crossbuild` command. It doesn't use any docker image.

I think we can make some improvements here. I should probably also add a `cgo` flag to the `crossbuild` command.
The thing is, currently both `build` and `crossbuild` are looking for the `build.cgo` conf because they both do actions based on this configuration.
I could probably add a `crossbuild.cgo` conf, but then we would need to specify both of this configurations in the `.promu.yml` because setting `crossbuild.cgo=true` without `build.cgo=ture` doesn't make sense.

I also thought about changing the `go` node in the yaml file (currently used to specify the golang version) to:

```yaml
go:
    version: 1.7
    cgo: true
```

But this would break existing `.promu.yml` usage that have a `go` version specified.